controllers/__init__.py

```diff
@@ -92,14 +92,27 @@ def upsert_evc(self, evc: Dict) -> Optional[Dict]:
         updated = self.db.evcs.find_one_and_update(
             {"_id": evc["id"]},
             {
-                "$set": model.dict(exclude={"inserted_at"}, exclude_none=True),
+                "$set": model.dict(
+                    exclude={"inserted_at"},
+                    exclude_none=True),
                 "$setOnInsert": {"inserted_at": utc_now},
             },
             return_document=ReturnDocument.AFTER,
             upsert=True,
         )
         return updated
 
+    def update_evc(self, evc: Dict) -> Optional[Dict]:
+        """Update or insert an EVC"""
```

Comment:
Docstring is confusing, can you clarify that this is just for updating? Also, for a double sanify could we also try to instantiate a `EVCBaseDoc(**evc)` but then still pass `evc` to `$set`? Currently, we have guarantees from the `validate_openapi` decorator, but if the api docs ended up diverging we might end up allowing more than what we needed, having an extra `EVCBaseDoc` here could help to avoid that both the models and the pre-guaranteed validation ever diverge.

Reply:
Got it. Added `EVCBaseDoc(**evc)` to check for any errors. Also change the docstring.